packages/sandbox/src/file_watching_sandbox.test.ts

```diff
@@ -216,8 +216,15 @@ void describe('Sandbox using local project name resolver', () => {
    * file change event function which tests can simulate by calling as desired.
    */
   beforeEach(async () => {
-    // ensures that .gitignore is set as absent
-    mock.method(fs, 'existsSync', () => false);
+    // ensures that rootDir only is exists and .gitignore is set as absent
```

Comment:
would it be better to return `p !== '.gitignore'` instead? Also applies to the mock above.

Reply:
Thanks. I think your suggestion is better.